src/coreComponents/unitTests/fluidFlowTests/testThermalCompMultiphaseFlow.cpp

```diff
@@ -44,193 +44,195 @@ char const * pvtGas = "DensityFun SpanWagnerCO2Density 1e6 7.5e7 5e5 295.15 370.
-  "phaseRelPermMaxValue=\"{ 0.9, 0.9 }\"/>\n"
-  "<MultiPhaseConstantThermalConductivity\n"
-  "name=\"thermalCond\"\n"
-  "phaseNames=\"{ gas, water }\"\n"
-  "thermalConductivityComponents=\"{ 0.6, 0.6, 0.6 }\"/>\n"
-  "</Constitutive>\n"
-  "<FieldSpecifications>\n"
-  "<FieldSpecification\n"
-  "name=\"initialPressure\"\n"
-  "initialCondition=\"1\"\n"
-  "setNames=\"{ all }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"pressure\"\n"
-  "scale=\"9e6\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"initialTemperature\"\n"
-  "initialCondition=\"1\"\n"
-  "setNames=\"{ all }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"temperature\"\n"
-  "scale=\"368.15\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"initialComposition_co2\"\n"
-  "initialCondition=\"1\"\n"
-  "setNames=\"{ all }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"globalCompFraction\"\n"
-  "component=\"0\"\n"
-  "scale=\"0.1\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"initialComposition_water\"\n"
-  "initialCondition=\"1\"\n"
-  "setNames=\"{ all }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"globalCompFraction\"\n"
-  "component=\"1\"\n"
-  "scale=\"0.9\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"sinkPressure\"\n"
-  "setNames=\"{ sink }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"pressure\"\n"
-  "scale=\"7e6\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"sinkTemperature\"\n"
-  "setNames=\"{ sink }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"temperature\"\n"
-  "scale=\"368.15\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"sinkTermComposition_co2\"\n"
-  "setNames=\"{ sink }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"globalCompFraction\"\n"
-  "component=\"0\"\n"
-  "scale=\"0.1\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"sinkTermComposition_water\"\n"
-  "setNames=\"{ sink }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"globalCompFraction\"\n"
-  "component=\"1\"\n"
-  "scale=\"0.9\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"sourcePressure\"\n"
-  "setNames=\"{ source }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"pressure\"\n"
-  "scale=\"1.45e7\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"sourceTemperature\"\n"
-  "setNames=\"{ source }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"temperature\"\n"
-  "scale=\"300.15\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"sourceTermComposition_co2\"\n"
-  "setNames=\"{ source }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"globalCompFraction\"\n"
-  "component=\"0\"\n"
-  "scale=\"0.9\"/>\n"
-  "<FieldSpecification\n"
-  "name=\"sourceTermComposition_water\"\n"
-  "setNames=\"{ source }\"\n"
-  "objectPath=\"ElementRegions/region/cb\"\n"
-  "fieldName=\"globalCompFraction\"\n"
-  "component=\"1\"\n"
-  "scale=\"0.1\"/>\n"
-  "</FieldSpecifications>\n"
-  "</Problem>\n";
+  R"xml(
+  <Problem>
```

Comment:
Is it OK for you to indent this `xml` content?
That would spare us an additional PR.
Any formatter will do 😉